app/tasks/education_form/create_daily_spool_files.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+module EducationForm
+  class CreateDailySpoolFiles
```

Comment:
@austin-martinez I put in in a `tasks` folder only because it seems like the terminology we've gone with is "tasks"

@saneshark right, the idea is to have a static kick-off time, and then let sidekiq/resque/whatever handle the retry logic as-needed. The whole sidekiq/resque/etc.. is another larger conversation we need to have as a team..